source/fixed_array.rb

```diff
@@ -1,3 +1,24 @@
 class FixedArray
+  attr_accessor :size, :array
 
+  def initialize(size, array)
+    @size = size
+    @array = Array.new(size, array)
+  end
+
+  def get(index)
+    if index <= self.size
+      self.array[index]
```

Comment:
Does `@array` need to be public? Could this be a private accessor?

Part of the point of FixedArray is to restrict you, so I'd be inclined to say no one outside the object should be able to access the ruby array underneath.